doc/stdlib/arrays.rst

```diff
@@ -80,117 +51,134 @@ largest range along each dimension.
-   element initialized to 1.0.
+   Create an array filled with the value ``x``. For example, element initialized to 1.0. If ``x`` is an object reference, all elements will refer to the same object. ``fill(Foo(), dims)`` will return an array filled with the result of evaluating ``Foo()`` once.
 
-   If ``x`` is an object reference, all elements will refer to the same object.
-   ``fill(Foo(), dims)`` will return an array filled with the result of evaluating ``Foo()`` once.
 
 .. function:: fill!(A, x)
 
-   Fill array ``A`` with the value ``x``. If ``x`` is an object reference, all elements will refer to the same object.
-   ``fill!(A, Foo())`` will return ``A`` filled with the result of evaluating ``Foo()`` once.
+   Fill array ``A`` with the value ``x``. If ``x`` is an object reference, all elements will refer to the same object. ``fill!(A, Foo())`` will return ``A`` filled with the result of evaluating
+
 
 .. function:: reshape(A, dims)
 
    Create an array with the same data as the given array, but with different dimensions. An implementation for a particular type of array may choose whether the data is copied or shared.
 
+
 .. function:: similar(array, element_type, dims)
 
-   Create an uninitialized array of the same type as the given array, but with the specified element type and dimensions. The second and third arguments are both optional. The ``dims`` argument may be a tuple or a series of integer arguments. For some special ``AbstractArray`` objects which are not real containers (like ranges), this function returns a standard ``Array`` to allow operating on elements.
+   Create an uninitialized array of the same type as the given array, but with the specified element type and dimensions. The second and third arguments are both optional. The ``dims`` argument may be a tuple or a series of integer arguments. For some special ranges), this function returns a standard ``Array`` to allow operating on elements.
+
 
 .. function:: reinterpret(type, A)
 
-   Change the type-interpretation of a block of memory. For example, ``reinterpret(Float32, UInt32(7))`` interprets the 4 bytes corresponding to ``UInt32(7)`` as a ``Float32``. For arrays, this constructs an array with the same binary data as the given array, but with the specified element type.
+   Change the type-interpretation of a block of memory. For example, corresponding to ``UInt32(7)`` as a ``Float32``. For arrays, this constructs an array with the same binary data as the given array, but with the specified element type.
+
+
+.. function:: eye(A)
+
+   Constructs an identity matrix of the same dimensions and type as
 
-.. function:: eye(n)
 
-   n-by-n identity matrix
+.. function:: eye(A)
 
-.. function:: eye(m, n)
+   Constructs an identity matrix of the same dimensions and type as
 
-   m-by-n identity matrix
 
 .. function:: eye(A)
 
-   Constructs an identity matrix of the same dimensions and type as ``A``.
+   Constructs an identity matrix of the same dimensions and type as
```

Comment:
This appears to be missing the remainder of the line (after a \`\`... not sure if what's the pattern, but there's quite a few lines below which demonstrate similar - the diff should definitely be combed through).

Also this (`function eye(A)`) now appears three times as the same signature :s